src/alire/alire-platforms.ads

```diff
@@ -62,4 +70,15 @@ package Alire.Platforms with Preelaborate is
 
    type Shells is (Unix, PowerShell, WinCmd);
 
+private
+
+   function Arch_Mapping (Arch : Extended_Architectures) return Architectures
+   is (case Arch is
```

Comment:
I think this could be:
```ada
 function Arch_Mapping (Arch : Extended_Architectures) return Architectures
   is (case Arch is
          when ARM64  => AARCH64,
          when Architectures => Arch
          when others =>
             raise Program_Error
               with "Mapping missing for given architecture: " & Arch'Image;
```

Reply:
Indeed, neat!